docs/sources/mimir/configure/configure-native-histograms-ingestion.md

```diff
@@ -7,7 +7,7 @@ weight: 160
 
 # Configure native histograms
 
-Prometheus native histograms ingestion is an experimental feature of Grafana Mimir.
+Native histograms are experimental and subject to change.
```

Comment:
In the [Helm chart's documentation about native histogram][1], we mark this same one with a note block. Maybe we could do that consistently everywhere.

```suggestion
{{% admonition type="note" %}}
Native histograms are experimental and subject to change.
{{% /admonition %}}
```

[1]: https://grafana.com/docs/helm-charts/mimir-distributed/latest/configure/configure-native-histograms-ingestion/

Reply:
agreed to copy from Helm